.github/scripts/update_googleapis_commit.sh

```diff
@@ -79,9 +84,16 @@ changed_files=$(git diff --cached --name-only)
 if [[ "${changed_files}" == "" ]]; then
     echo "The latest googleapis commit is not changed."
     echo "Skip committing to the pull request."
+else
+    git commit -m "${title}"
+fi
+
+unpushed_commit=$(git cherry -v "origin/${current_branch}" | wc -l)
```

Comment:
What is this unpushed commit check meant to ensure? I'm not familiar enough with the cherry-pick subcommands

Reply:
After this change, we may have up to two commits: merge commit and change commit.

We want to exit the script if no commit happens (otherwise this will be a infinite loop). `git cherry` is a way to find whether the local branch has commits that are not in the remote branch. If we find any such commit, push them to remote branch.

Reply:
Makes sense, thanks. Maybe a small comment in this line may help others as well on how this ensures the remote is synced?